katana/native/file_manager/file_manager_utils/backend.py

```diff
@@ -0,0 +1,121 @@
+#!/usr/bin/python3
+import os
+import shutil
+from katana.utils import user_utils
+
+def getpath(request):
+    """
+    This function is to get the path of the User-data location.
+    Whatever is pointed to by this path is displayed.
+    Currently, a static path is read from the file in my VM and the contents are displayed.
+    The file is opened and the path is read
+
+    :return: The path to user-data space
+    """
+    Object = user_utils.UserData('File Manager')
+    print(Object.get_user_path(request))
+    return Object.get_user_path(request)
+
+def delete(path):
+    """
+    This will check if it's a file or directory and then delete them.
+
+    :param path: Full path to File or Directory which has to be deleted.
+    :return: A string saying success or error
+    """
+    if (os.path.isdir(path)):
+        try:
+            shutil.rmtree(path)
+            return 'success'
+        except OSError:
+            print('Cannot delete Directory')
+    elif (os.path.isfile(path)):
+        try:
+            os.remove(path)
+            return 'success'
+        except OSError:
+            print('Cannot delete File')
```

Comment:
Please return something in except blocks as well. This is make the code cleaner. Please use one return at the end of the function call rather than having 3-4 separate return statements.

Reply:
Done